openedx/core/djangoapps/ace_common/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -62,7 +62,7 @@
                     <tr>
                         <td width="70">
                             <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://media.sailthru.com/595/1k1/8/o/599f355101b3f.png" width="70"
+                                    src="https://edx-cdn.org/v3/prod/logo.png" width="70"
```

Comment:
Probably a silly question, but why do we use this template in edx-platform as opposed to an override in edx-themes?

Reply:
Good call, we could have overridden it in the first place. unfortunately, I am not aware of what was the decision at that time. I think it's easier, for now, to keep using the template here and think about overriding it post 15dec.

Reply:
@awaisdar001 Can you create JIRA tickets for post-Dec-15th cleanup work to ensure we come back to outstanding issues that weren't addressed within the Dec-15th time crunch?

Reply:
Sure.